src/vignette/util/thumbnail.clj

```diff
@@ -29,6 +32,9 @@
                   :window-width "window-width"
                   :window-height "window-height"})
 
+(def mime-blacklist #{"audio/ogg"
```

Comment:
Instead of `mime-blacklist` should this be `unsupported-mime-types` or something similar that indicates more clearly that we aren't supporting thumbnail operations on these types?